contrib/kubernetes/manifests/registry-s3-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-keyfile
+type: Opaque
+data:
+  accesskey: {base64 encode of the access key}
+  secretkey: {base64 encode of the secret key}
+  region: {base64 encode of the region name}
+  bucket: {base64 encode of the buket name}
```

Comment:
I think having everything in a single file is better as it easy to make changes in one place rather than multiple places.
https://github.com/deis/charts/issues/95 will take care of templating all the charts.